expression/builtin_json_vec.go

```diff
@@ -448,10 +448,14 @@ func (b *builtinJSONSearchSig) vecEvalJSON(input *chunk.Chunk, result *chunk.Col
 				pathExprs = append(pathExprs, pathExpr)
 			}
 		}
-		bj, _, err := jsonBuf.GetJSON(i).Search(containType, searchBuf.GetString(i), escape, pathExprs)
+		bj, isNull, err := jsonBuf.GetJSON(i).Search(containType, searchBuf.GetString(i), escape, pathExprs)
```

Comment:
Could you please add a unit test for this?

Reply:
Any suggestions or guides for where to properly add this unit test? The origin presentation of this issue is JSON_SEARCH, but cause of this is from expression.

Reply:
PTAL @SunRunAway

Reply:
1) Add cases [here](https://github.com/pingcap/tidb/blob/7cc2c700414d0ba6a71115538d3fcf523dfd40a1/expression/builtin_json_vec_test.go#L115), for comparing the results of `evalJSON` and `vecEvalJSON`.
2) integration test also can be added in [integration_test.go](https://github.com/pingcap/tidb/blob/5eaaa5c64909bd9f3833209e6b0d974d31f9917a/expression/integration_test.go)

Reply:
PTAL